client/ayon_maya/plugins/publish/validate_animation_rig_content.py

```diff
@@ -0,0 +1,40 @@
+import inspect
+from ayon_core.pipeline.publish import (
+    OptionalPyblishPluginMixin,
+    PublishValidationError,
+    ValidateContentsOrder,
+)
+from ayon_maya.api import plugin
+
+
+class ValidateAnimatedRigContent(plugin.MayaInstancePlugin,
+                                 OptionalPyblishPluginMixin):
+    """Validates the `skeletonAnim_SET` must have one or more objects
+    """
+    order = ValidateContentsOrder + 0.05
+    label = "Animated Rig Content"
+    families = ["animation.fbx"]
+    optional = True
```

Comment:
Out of curiosity - why is this optional? :) Doesn't the extractor fail if skeleton anim set is empty?

Reply:
I meant more - if it's a required check and it should fail otherwise.. It shouldn't be **optional** but be a required validation. Correct?

Reply:
let's not be optional.

Reply:
Nevermind - it's non optional in settings - I'll leave it at that.